vignettes/SKG2020_submission/skg2020_submission.Rmd

```diff
@@ -103,52 +124,55 @@ While writing the package, we collected exemplary OpenAIRE Research Graph record
 
 A main purpose of the OpenAIRE Research Graph, which links grants to publications and open access full-texts, is monitoring the efficacy of the open access mandates. Here, we will demonstrate how to benchmark the compliance with the open access mandate of the European Commission’s HORIZON 2020 funding programme (H2020) against related funding activities using our implementation. We will focus on projects affiliated with the University of Göttingen. In doing so, we want to take into account that open access mandates generally lead to above-average open access uptake levels. Yet, they can vary by discipline and funding activity [@Larivi_re_2018].
 
-```{r, cache = TRUE}
-library(openairegraph)
-library(jsonlite)
-library(tidyverse)
-library(scales)
-library(here)
```

Comment:
we only need this once for the entire vignette/article. `library()` calls persist.
The danger here, though is, that the chunk in question is cached.
Caches, unfortunately, don't carry side effects, so we should place whatever has side effects (such as `library()` inside non-cached chunks).